### PR TITLE
Ports Knight Armor

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Ports: https://github.com/tgstation/-tg-station/pull/13220

Adds in a bunch of knight armor--primarily offers moderate melee protection. Chaplain gets the knight's templar armor for thematic+RP reasons and also so he actually stands a fighting chance against his mortal enemies: the cult, vampires, and wizards.

Also ports over the ERT paranormal knight armor; has built in rune protection against cultists (by virtue of the null rod).

pics.

Red, Blue, Green, and Yellow Knight Armor:
img1
img2
img3
img4
Chaplain's Crusade Knight Armor:
img5
Paranormal ERT Armor:
img6

Null Rod Conversion
- The Chaplain can sheath his null rod into the crusader armor to transform it into a holy sword
 - This is just a reskinned null rod with a higher weight class; it's mostly just so it's thematically coherent with the crusader armor. 

img6

Chaplain's Closet
- Chaplain's Closet is now a secure closet
 - No, you do not get to steal the chaplain's stuff anymore

img7
sprites for the closet by VG

:cl: Fox McCloud
rscadd: Adds in knight armor
rscadd: Grants the chaplain a set of crusader knight armor
rscadd: Adds in cult-resistant ERT paranormal space suit
rscadd: Allows the Chaplain to convert his null rod into a holy sword with crusader knight armor
tweak: Chaplain's closet is now a secure closet
/:cl:

### CODE REVIEW

**Comment by NTSAM** [comment]
PRAISE THE SUN

**Comment by CrAzYPiLoT-SS13** [comment]
Deus vult!

**Comment by Plotron** [comment]
I like the idea.
What about the standard armor sets? Are they possible to get anywhere?

**Comment by Crazylemon64** [comment]
Oh man, now I can finally play a paladin!

**Comment by TheDZD** [comment]
Does that paranormal ERT armor remind anyone else of [something specific](http://vignette3.wikia.nocookie.net/warhammer40k/images/d/d0/GreyKnightTriumphant.jpg/revision/latest?cb=20120604204351)?

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -154,6 +154,35 @@ obj/item/clothing/head/blob
 	flags = HEADCOVERSEYES|HEADCOVERSMOUTH
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES
 
+/obj/item/clothing/head/helmet/knight
+	name = "medieval helmet"
+	desc = "A classic metal helmet."
+	icon_state = "knight_green"
+	item_state = "knight_green"
+	armor = list(melee = 41, bullet = 15, laser = 5,energy = 5, bomb = 5, bio = 2, rad = 0)
```
Comment (TheDZD) [inline code/modules/clothing/head/helmet.dm]: Some of the values here are oddly specific, and they don't match up with the armor oddly enough.

**Comment by ScratKnapp** [comment]
How much armor does the crusader armor provide? Im shit at reading code.

**Comment by Fox-McCloud** [comment]
50 melee and minor armor for other things.
Helmet is 41 with minor armor on other things.

**Comment by Fox-McCloud** [comment]
So it occurred to me, having a null rod with the crusader armor was rather silly, so I added a way to convert the nullrod into a holy sword; it's just a reskinned null rod that has a higher weight class.

**Comment by Fox-McCloud** [comment]
Anddd because I don't want people breaking in for a set of armor, the chaplain's closet is now a secure closet.

**Comment by Plotron** [comment]
Holy *shit*sword, fantastic idea.
I see in code that the sword gains a sharp flag.
That would mean it's slightly more robust than the null rod, provided you've got a place to store it (increased weight).

**Pull request merged**